docs/interfaces/accelerometer.md

```diff
@@ -0,0 +1,26 @@
+# Accelerometer
```

Comment:
we should add more info about accelerometer. for hmx to get new data sampled continues you will need to setup a special command over i2c-tools.